doc/api/events.md

```diff
@@ -1102,7 +1102,7 @@ In either case, the handler function will be invoked with the `event` argument
 passed to the `eventTarget.dispatchEvent()` function.
 
 Async functions may be used as event listeners. If an async handler function
-rejects, the rejection will be captured and be will handled as described in
+rejects, the rejection will be captured and will be handled as described in
```

Comment:
Alternative (optional) suggestion to avoid double "will be"s:

```suggestion
rejects, the rejection will be captured and handled as described in
```